readthedocs/api/v2/models.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta
+
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+from rest_framework_api_key.models import AbstractAPIKey, BaseAPIKeyManager
+
+from readthedocs.projects.models import Project
+
+
+class BuildAPIKeyManager(BaseAPIKeyManager):
+    def create_key(self, project):
+        """
+        Create a new API key for a project.
+
+        Build API keys are valid for 3 hours,
+        and can be revoked at any time by hitting the /api/v2/revoke/ endpoint.
+        """
+        expiry_date = timezone.now() + timedelta(hours=3)
```

Comment:
The other day I learned that you can do `timezone.timedelta()`. So there is no need to do `from datetime import timedelta` 😉

Reply:
timedelta is imported from datetime in Django, it isn't part of the module itself https://github.com/django/django/blob/649262a406168709686f97694493aa1f717c6c96/django/utils/timezone.py#L14-L30, we shouldn't rely on implicit re-imports.

Reply:
Interesting... Good point.